en/docs/apis/create-and-publish-your-first-api.md

```diff
@@ -3,43 +3,41 @@
 Choreo’s API management capabilities allow you to create, publish, and manage all aspects of an API and its lifecycle.
 This quick start guide walks you through the steps to quickly create and publish an API using Choreo. 
 
-## Prerequisites:
+## Prerequisites
 - A mock REST service to use when creating the API. 
     
     !!!note
-        In this quick start guide, let's use the mock service [http://www.mocky.io/v2/5185415ba171ea3a00704eed](http://www.mocky.io/v2/5185415ba171ea3a00704eed) that is made available by [https://www.mocky.io/](https://www.mocky.io/). Click the mock service URL to test the service. You can see the following JSON message:`{"hello": "world"}`
+        In this quick start guide, let's use the mock service at [http://www.mocky.io/v2/5185415ba171ea3a00704eed](http://www.mocky.io/v2/5185415ba171ea3a00704eed), which is made available by [https://www.mocky.io/](https://www.mocky.io/). Click the mock service URL to test the service. You'll see the following JSON message: `{"hello": "world"}`.
  
 ## Step 1: Create an API from scratch
-Follow the procedure below to create an API using the mock REST service you tested in the prerequisites section:
+Follow this procedure to create an API using the mock REST service you tested in the prerequisites section:
 
-1. Access the Choreo Console via [https://console.choreo.dev/](https://console.choreo.dev/).
-2. Sign in using either your GitHub or Google credentials.
-3. Go to the **APIs** card and click **Get Started**. This takes you to the **Create API** page. 
+1. Access the Choreo Console via https://console.choreo.dev/ and sign in using your Google or GitHub credentials.
+2. Go to the **APIs** card and click **Get Started**. This takes you to the **Create API** page. 
     
     !!!note
         If this is not the first API you are creating, you must click **+ Create** to go to the **Create API** page.
 
-4. On the **Create API** page, go to the **Start from scratch card** and click **Next**.
-5. In the **Create API from REST API** form, fill in the details as follows:
+3. On the **Create API** page, go to the **Start from scratch card** and click **Next**.
+4. In the **Create API from REST API** form, fill in the details as follows:
     - **API Name**: `HelloWorld`
```

Comment:
shouldn't the user input be in bold? https://docs.microsoft.com/en-us/style-guide/procedures-instructions/formatting-text-in-instructions

Reply:
Although the copy editor suggested bold to format user input values, we want to suggest the use of backticks because it can be confusing when the field names and user input values are formatted in the same way.

Reply:
+1 to have the user input values formatted with black ticks for the same reasons given by Nashaath.